src/core/providers/ogr/qgsogrprovidermetadata.cpp

```diff
@@ -1028,6 +1029,64 @@ bool QgsOgrProviderMetadata::uriIsBlocklisted( const QString &uri ) const
   return false;
 }
 
+QList<QgsProviderSublayerDetails> QgsOgrProviderMetadata::querySublayers( const QString &uri, Qgis::SublayerQueryFlags flags, QgsFeedback *feedback ) const
+{
+  QStringList skippedLayerNames;
+
+  QStringList options { QStringLiteral( "@LIST_ALL_TABLES=YES" ) };
+
+  QString errCause;
+  QgsOgrLayerUniquePtr firstLayer = QgsOgrProviderUtils::getLayer( uri, false, options, 0, errCause, true );
+  if ( !firstLayer )
+    return {};
+
+  const QString driverName = firstLayer->driverName();
+  if ( driverName == QLatin1String( "SQLite" ) )
+  {
+    skippedLayerNames = QgsSqliteUtils::systemTables();
```

Comment:
skippedLayerNames is not used after wards

Reply:
Fixed